src/orchestration.d.ts

```diff
@@ -5,16 +5,32 @@ import { Delegation, Redelegation, UnbondingDelegation } from './delegation.js';
 import type { Invitation } from '@agoric/zoe/exported.js';
 import type { Amount, Brand, Payment, Purse } from '@agoric/ertp/exported.js';
 
-/** static declaration of known chain types will allow type support for
+/**
+ * Static declaration of known chain types will allow type support for
  * additional chain-specific operations like `liquidStake`
  */
 export type KnownChains = {
-  stride: ChainInfo;
-  cosmos: ChainInfo;
-  agoric: ChainInfo;
-  celestia: ChainInfo;
-  osmosis: ChainInfo;
-  // ...
+  stride: {
+    info: CosmosChainInfo;
+    methods: {
```

Comment:
nice solution. I expect we'll have mixin types later but this is a good direction